spec/javascripts/health-care/components/Nav.spec.jsx

```diff
@@ -57,17 +57,14 @@ describe('<Nav>', () => {
 
     const expectOneActiveLink = (component, path) => {
       history.replace(path);
-      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'usa-current');
+      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'section-current');
       expect(activeLinks).to.have.lengthOf(1);
-      expect(activeLinks[0].getAttribute('href')).to.equal(path);
     };
 
     const expectActiveLinksForNavAndSubNav = (component, path) => {
       history.replace(path);
-      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'usa-current');
+      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'section-current');
       expect(activeLinks).to.have.lengthOf(2);
-      expect(activeLinks[0].getAttribute('href').startsWith(activeLinks[1].getAttribute('href')));
```

Comment:
That sounds fine to me. Also, I don't know if we are planning to send an object for routes so they build dynamically but that might be down the road.